src/components/ListItem.jsx

```diff
@@ -70,7 +69,7 @@ export function ListItem({ item, listId }) {
 				{item.name} {deleteError}
 			</label>
 
-			<li>Purchase again: {purchaseUrgencyMessage(item)}</li>
+			<span>Purchase again: {purchaseUrgencyMessage(item)}</span>
```

Comment:
I think this is the correct semantic choice for displaying this info, and the li element within the other li element was rendering an error on the dev browser console.